.github/workflows/build.yml

```diff
@@ -21,7 +21,7 @@ jobs:
         with:
           path: "~/.cache/bazel"
           key: bazel
-      - name: Build
-        run: bazel build ...
+      - name: Build and test
+        run: bazel test ...
```

Comment:
Does `bazel test` build everything, or just the parts that are covered by tests?

Reply:
Everything. See [Bazel test docs](https://bazel.build/docs/user-manual#running-tests):

> ![image](https://user-images.githubusercontent.com/961451/215504992-6313cf62-bf41-4d44-b135-cabf86a18fb9.png)